signer/src/lib.rs

```diff
@@ -37,6 +37,11 @@ pub mod ecdsa;
 #[cfg_attr(docsrs, doc(cfg(feature = "unstable-eth")))]
 pub mod eth;
 
+/// A polkadot-js account json loader.
+#[cfg(feature = "json")]
```

Comment:
`json` feels a bit vague as a feature name to me; perhaps `"polkadot-js-json-compat"` or similar makes more sense, to be clear that this is about decoding the specific polkadot-js JSON format?

Reply:
done